lib/config/config-file.js

```diff
@@ -392,6 +392,16 @@ function applyExtends(config, filePath, relativeTo) {
  * @private
  */
 function normalizePackageName(name, prefix) {
+
+    /*
+     * On Windows, name can come in with Windows slashes instead of Unix slashes.
+     * Normalize to Unix first to avoid errors later on.
+     * https://github.com/eslint/eslint/issues/5644
+     */
+    if (name.indexOf("\\") > -1) {
+        name = name.replace(/\\/g, "/");
```

Comment:
We already have function exposed in utils doing similar stuff: https://github.com/eslint/eslint/blob/master/lib/util/path-util.js#L26
I think we should use that function here.